### ISSUE TITLE
Reproducing MRA-BGCN on METR-LA

### ISSUE THREAD

**Issue by trainingl** [j1o2h3n/MRA-BGCN#2]
Thanks for your excellent work! I am very encouraged by the edge graphs in the paper and was wondering if you have reproduced similar results to the original MRA-BGCN paper on the METR-LA dataset?

**Comment by j1o2h3n** [comment]
Sorry, I didn't run the code on the METR-LA dataset.

**Comment by trainingl** [comment]
I have a key question about the application of MRA-BGCN: the edge-wise subgraph constructed in the paper or code has a tensor size of (num_edges, num_edges), and in the traffic speed dataset or data with more edges, since the number of edges is much larger than the number of nodes, will this bring serious computational overhead and memory cost when calculating graph convolution?

**Comment by j1o2h3n** [comment]
Yes, I agree with you, the number of edges is more dense compared to nodes, which introduces more calculation parameters into the model, and the additional resource consumption is not optimistic.
